docs/docs-beta/docs/dagster-plus/deployment/management/managing-compute-logs-and-error-messages.md

```diff
@@ -3,4 +3,102 @@ title: Managing compute logs and error messages
 unlisted: true
 ---
 
-{/* TODO move from https://docs.dagster.io/dagster-plus/managing-deployments/controlling-logs */}
\ No newline at end of file
+import ThemedImage from '@theme/ThemedImage';
+
+:::note
+This guide is applicable to Dagster+.
+:::
+
+In this guide, we'll cover how to adjust where Dagster+ compute logs are stored and manage masking of error messages in the Dagster+ UI.
+
+By default, Dagster+ ingests [structured event logs and compute logs](/concepts/logging#log-types) from runs and surfaces error messages from [code locations](/guides/deploy/code-locations/) in the UI.
```

Comment:
A non Dagster+ reference to `/guides/deploy/code-locations/` does not seem to exist.